src/components/overview-table.js

```diff
@@ -131,6 +127,7 @@ function FileDataRow({ processFile, availableInputs, fileGroup, timestamp }) {
 
     const handleClose = () => {
         setOpen(false);
+        setSelectedFilename('');
     };
 
     const selectFile = useCallback(
```

Comment:
If you reset the filename in the `handleClose` function, shouldn't you call `handleClose` *after* `fetchTaskManagerSelectFile`?